driver/conv_driver.hpp

```diff
@@ -594,6 +631,12 @@ int ConvDriver<Tgpu, Tref>::GetandSetData()
     SetConvDescriptorFromCmdLineArgs();
 
     std::vector<int> out_len = GetOutputTensorLengths();
+    if(miopen::deref(inputTensor).GetLayout_t() == miopenTensorNCHWc4 ||
+       miopen::deref(inputTensor).GetLayout_t() == miopenTensorNCHWc8)
+        out_len[1] *= miopen::deref(inputTensor).GetVectorLength();
```

Comment:
Thanks for the suggestion. I'm not sure what "updating" actually mean: change or add the` .GetVectorLength() != 1`? We can not abandon the tensor layout related condition, because the vectorized dimension is at different position in NCHWc and CHWNc tensors.

Reply:
Sorry, they just look very simmilar at fast glance.